net/rpc/server_test.go

```diff
@@ -449,8 +444,6 @@ func (codec *CodecEmulator) Close() error {
 }
 
 func TestServeRequest(t *testing.T) {
-	once.Do(startServer)
-	testServeRequest(t, nil)
```

Comment:
This was testing the `ServeRequest` (pacakage-level function that used `DefaultServer`). That is being removed, so there's no longer any need to test it. 

You can see that being tested in the diff just above this line (line 412-417 in this file).